enterprise/frontend/src/metabase-enterprise/settings/selectors.ts

```diff
@@ -18,6 +19,10 @@ export const getLogoUrl = (state: EnterpriseState) =>
 export const getLoadingMessage = (state: EnterpriseState) =>
   LOADING_MESSAGE_BY_SETTING[getSetting(state, "loading-message")];
 
-const DEFAULT_APPLICATION_NAME = "Metabase";
-export const getIsWhiteLabeling = (states: EnterpriseState) =>
-  getSetting(states, "application-name") !== DEFAULT_APPLICATION_NAME;
+const DEFAULT_APPLICATION_NAME = t`Metabase`;
```

Comment:
Should't this always be "Metabase"?

Reply:
Possibly I did that because I saw we translate `Metabase` in some places, but I forgot we must have done that because we want to replace it with the user app name (whitelabel) because we hack that with `ttag`

Reply:
Fixed in https://github.com/metabase/metabase/pull/36032/commits/141e4813e11602fdaa60bbb6923000a9c7d89941